### PR TITLE
Figure Updates

### PR DESCRIPTION
**Pull request by JacksonLChin**
Various updates:
- Wrapper for CMTF method
- Removing old figures
- Standardizing import statements
- Updating figures and removing old models

### CODE REVIEW

**Comment by JacksonLChin** [comment]
@aarmey @cyrillustan  Running into some issues here with  the CMTF wrapper. I've placed the wrapper function, _run_CMTF_, in dataImport and divide the mFactor by variance to normalize our components. This is leading to some issues Figure 2b since modifying the CMTF mFactor messes with the R2X computations.

**Comment by aarmey** [comment]
@JacksonLChin maybe it's easier to only normalize mFactors when you plot it, then?

File: tfac/figures/figure2.py [hunk 1]
```diff
@@ -3,10 +3,11 @@
 """
 import numpy as np
 import pandas as pd
-from .common import getSetup
-from ..dataImport import form_tensor
-from ..predict import evaluate_accuracy
-from tensorpack import perform_CMTF, calcR2X
+
+from tfac.dataImport import run_CMTF
+from tfac.figures.common import getSetup
```
Comment (aarmey) [inline tfac/figures/figure2.py:8]: Is there a reason you need to do these all as absolute imports, and not relative?

**Review by aarmey — COMMENTED** [review]
Just one question so far.

**Comment by aarmey** [comment]
@JacksonLChin also look at Fig. 2. Why are you getting R2X = 1 in the scaling plot?

**Comment by aarmey** [comment]
@JacksonLChin I think I would rather review this broken up into individual changes. For example, *just* update tensorpack and make sure the results don't change when the gene modules are normalized. Then, *just* swap out the modules in a separate PR. There are too many things changing at once here, and so it is extremely difficult to track down problems.

**Comment by JacksonLChin** [comment]
Sounds good. A lot of changes came in so this PR became a bit of an omnibus. I'll close this PR and open up a few separate ones.

**Pull request closed without merging**